tests/test-helper.ts

```diff
@@ -16,5 +16,7 @@ faker.seed(17);
     // This ensures all engine resources are loaded before the tests
     await preloadAssets(manifest);
 
-    start();
+    start({
+        setupTestIsolationValidation: true,
```

Comment:
I've checked the ember-qunit docs to understand this option, but I'm still unsure what this option does. Could you explain what this option does, for the layman?

Reply:
[This](https://github.com/emberjs/ember-qunit/blob/57c3327d1e3402b92afbfa067ab91d5428817807/docs/TEST_ISOLATION_VALIDATION.md) explains it best. Merely paraphrasing. My understanding is that this flag allows us to figure out async leakage in tests. Ideally, when a test finishes running there should not be any pending requests/async operation as that might affect the next test. So if you schedule (on the run loop, there is only one per application instance) a task or function to run after some duration, say using `Ember.run.later`, there is a possibility that at some point after this test has finished, the function/task will run hence interfering with the current test. We want to avoid this sort of thing as they are probabilistic and hard to debug. This flag allows to figure out if any of tests leaks by detecting if the test has any
`pending AJAX requests`, `pending test waiters`, `scheduled async`, `scheduled autorun` after completion.